th2_data_services_lwdp/provider_api/grpc.py

```diff
@@ -96,66 +89,36 @@ def search_events(
             end_timestamp: Sets the timestamp to which the search will be performed, starting with 'start_timestamp'.
                 Expected in nanoseconds.
             parent_event: Match events to the specified parent.
-            search_direction: Sets the lookup direction. Can be 'NEXT' or 'PREVIOUS'.
-            resume_from_id: The last event id from which we start searching for events.
-            result_count_limit: Sets the maximum amount of events to return.
-            keep_open: Option if the search has reached the current moment,
-                it is necessary to wait further for the appearance of new data.
-            limit_for_parent: How many children events for each parent do we want to request.
-            metadata_only: Receive only metadata (true) or entire event (false) (without attachedMessageIds).
-            attached_messages: Option if you want to load attachedMessageIds additionally.
-            filters: Which filters to apply in a search.
 
         Returns:
             Iterable object which return events as parts of streaming response.
         """
         self.__search_basic_checks(
             start_timestamp=start_timestamp,
-            end_timestamp=end_timestamp,
-            result_count_limit=result_count_limit,
-            search_direction=search_direction,
+            end_timestamp=end_timestamp
         )
 
         basic_request = self.__build_basic_request_object(
             start_timestamp=start_timestamp,
-            end_timestamp=end_timestamp,
-            search_direction=search_direction,
-            result_count_limit=result_count_limit,
-            keep_open=keep_open,
-            filters=filters,
+            end_timestamp=end_timestamp
         )
         parent_event = EventID(id=parent_event) if parent_event else None
-        resume_from_id = EventID(id=resume_from_id) if resume_from_id else None
-        limit_for_parent = Int64Value(value=limit_for_parent) if limit_for_parent else None
-        metadata_only = BoolValue(value=metadata_only)
-        attached_messages = BoolValue(value=attached_messages)
 
         event_search_request = EventSearchRequest(
             start_timestamp=basic_request.start_timestamp,
             end_timestamp=basic_request.end_timestamp,
-            parent_event=parent_event,
-            search_direction=basic_request.search_direction,
-            resume_from_id=resume_from_id,
-            result_count_limit=basic_request.result_count_limit,
-            keep_open=basic_request.keep_open,
-            limit_for_parent=limit_for_parent,
-            metadata_only=metadata_only,
-            attached_messages=attached_messages,
-            filter=basic_request.filters,
+            parent_event=parent_event
         )
         return self.__stub.searchEvents(event_search_request)
 
     def search_messages(
         self,
-        start_timestamp: int,
+        stream: List[str],
```

Comment:
Move, please, stream to previous place
Clients are used to putting start time, end time first.

Reply:
Changed it.